schema/course.model.schema

```diff
@@ -348,6 +348,28 @@
                   "type": "number",
                   "title": "Navigation bar order",
                   "default": 0
+                },
+                "_navTooltip": {
```

Comment:
I guess you'd add something like `_globals._extension._navigation._backTooltip` for the back button?